sunpy/visualization/visualization.py

```diff
@@ -36,6 +36,22 @@ def fn_itoggle(*args, **kwargs):
 
 
 def axis_labels_from_ctype(ctype, unit):
+    """
+    A function to return axis labels for the given coordinate type and unit
+    using python dictionary.
+    
+    Parameters
+    ----------
+    ctype: string
+        coordinates type
+    unit: string
+        required units
+    
+    Returns
+    -------
+    string
+        Axis_label [unit]
```

Comment:
You should expand on this.

Reply:
Should I give more explanation?

Reply:
Yes. Currently each sentence does not explain enough.

Reply:
Maybe they do not need more information.  For now just add capital letters and a full stop.